scripts/statistics/ci.js

```diff
@@ -1,65 +0,0 @@
-const fs = require('fs/promises');
```

Comment:
Looks good. Just a question about these statistics, why have they been deleted?

Reply:
Because we don't use them, most of the data is already available in LFX Insights platform and the results are being lost every three months or so.